pyproject.toml

```diff
@@ -63,18 +82,51 @@ markers = [
     "mymarker: example markers goes here"
 ]
 
+# ------------------ #
+# Ruff configuration #
+# ------------------ #
+# NOTE: We recommend to list the ruff configuration here and let pre-commit read it from here.
+#       This way, the configuration is consistent across all users.
 [tool.ruff]
+target-version = "py311"
 line-length = 120
 select = ["A", "ARG","ASYNC","BLE","C90", "E", "F", "I", "N", "UP032", "W"]
+exclude = ["notebooks/**", "**/*.ipynb"]
+
+[tool.ruff.lint]
+select = ["E", "F", "I"]
+ignore = []
 
-# Pixi configuration
+[tool.ruff.lint.isort]
+known-first-party = ["ibeatles"]
```

Comment:
Should this instead refer to `packagenamepy`?

Reply:
good catch, fixing it now